Sources/eppo/EppoClient.swift

```diff
@@ -45,39 +45,51 @@ public class EppoClient {
         self.flagConfigs = try JSONDecoder().decode(FlagConfigJSON.self, from: urlData);
     }
     
+    public func getAssignment(
+        _ subjectKey: String,
+        _ flagKey: String,
+        _ subjectAttributes: SubjectAttributes) throws -> String?
+    {
+        return try getStringAssignment(subjectKey, flagKey, subjectAttributes)
```

Comment:
I don't think we can just call `getStringAssignment()` because while similar these two do different things:
* `getAssignment()` should just return the old-school, string version of the assignment like we were doing before these typed methods.
* `getStringAssingent()` should return the string assignment _if it is a string_.

So if the assignment is a numeric assignment (e.g., `123`) then `getAssignment()` should return `"123"`, `getNumericAssignment()` should return `123`, and `getStringAssginment()` should return `nil`.

Since we want `getAssignment()` to work regardless of type, we'll want something like a `toString()` on `EppoValue` (which we'll need for logging allocations anyways) and should return that.

(Alternatively, we could also store the untyped value somewhere and just pass that through)

Reply:
Good point - I will return the contents of `variation.value` in this case.